Examples/DataRepresentation/Image/Image4.cxx

```diff
@@ -443,8 +442,7 @@ main(int, char *[])
   // Software Guide : BeginCodeSnippet
   using MatrixType =
     itk::Matrix<itk::SpacePrecisionType, Dimension, Dimension>;
-  MatrixType SpacingMatrix;
-  SpacingMatrix.Fill(0.0F);
+  auto SpacingMatrix = itk::MakeFilled<MatrixType>(0.0F);
```

Comment:
Again, no problem, but I would prefer `{}` for zero-initialization.

Reply:
Done.